packages/extension-ui/src/components/ScrollWrapper.tsx

```diff
@@ -5,44 +5,49 @@ import React from 'react';
 import styled from 'styled-components';
 
 import { ThemeProps } from '../types';
-import PasswordFeedback from './PasswordField/PasswordFeedback';
-import Address from './Address';
-import Label from './Label';
 
 interface Props extends ThemeProps {
   className?: string;
   children: React.ReactNode;
 }
 
 const ScrollWrapper: React.FC<Props> = ({ children, className }) => {
-  return <div className={className}>{children}</div>;
+  return (
+    <ScrollbarContainer className={className}>
+      <ScrollWrapperContent>{children}</ScrollWrapperContent>
+    </ScrollbarContainer>
```

Comment:
This comment isn't necessarily an issue with this line, I just wanted it to be a part of this batch of review comment, please, let me know if there is a better way :sweat_smile: 

Double padding?
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/6f5a7e09-6994-4d20-ba45-7ad46ff347ee)


height: 100% missing from ScrollWrapperContent?
also - bottom wrapper too narrow, I think
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/7e0c2a66-7726-48bb-a241-4f09aac540d2)


Header margin overflows content:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/26a576ab-a4bc-46dd-861b-2273d48a1c2e)

Reply:
Ok the header margin was obviously a bad idea - let me switch it to content margins.